worlds/messenger/__init__.py

```diff
@@ -1,10 +1,11 @@
-from typing import Dict, Any, List, Optional
+import logging
+from typing import Dict, Any, Optional
 
 from BaseClasses import Tutorial, ItemClassification
 from worlds.AutoWorld import World, WebWorld
-from .Constants import NOTES, PROG_ITEMS, PHOBEKINS, USEFUL_ITEMS, ALWAYS_LOCATIONS, SEALS, ALL_ITEMS
+from .Constants import *
```

Comment:
strongly recommending against `*` import. imo, the only proper use of those is when wrapping another file.